app/lib/application_form_status_updater.rb

```diff
@@ -84,6 +84,9 @@ def new_status
         "waiting_on"
       elsif received_further_information || received_qualification
         "received"
+      elsif application_form.region.requires_preliminary_check &&
```

Comment:
I think we should copy the value of `requires_preliminary_check` to a field on the application form when creating one rather than checking the value of the region. This should avoid problems where an application form already exists and we then enable/disable value on the region. This is similar to what we do for [some other fields](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/cf7bed35f5e803945945b2aa8be8bbafb5e0d012/app/lib/application_form_factory.rb#L15-L19).

Reply:
Makes sense, it would certainly produce odd side-effects. Have appended commits to do this, can squash these down if needed.